### ISSUE TITLE
prediction for matter-matter correlation not being smoothed

### ISSUE THREAD

**Issue by cramirezpe** [damonge/CoLoRe#64]
Is there a reason for the matter-matter correlation (but also galaxy-matter) prediction not being smoothed? 

In predictions.c we find:
https://github.com/damonge/CoLoRe/blob/0460c20a9a71194b213f11469fbb4e637bc66307/src/predictions.c#L92-L99

That means that we are saving to file the smoothed power spectrum and the linear (not smoothed) correlation. Is there a reason for that, or it is just an error?

**Comment by damonge** [comment]
Yep, you're absolutely right. I didn't implement that because I wasn't using the correlation functions for anything.
